tarorpc/taro.proto

```diff
@@ -123,6 +129,32 @@ enum AssetType {
     */
     COLLECTIBLE = 1;
 }
+
+enum AssetMetaType {
+    /*
+    Opaque is used for asset meta blobs that have no true structure and instead
+    should be interested as opaque blobs.
+    */
+    OPAQUE = 0;
```

Comment:
nit: all enum values are in the same namespace that doesn't include the enum name itself. So we couldn't have another value called `OPAQUE` in a different enum. We should prefix the value, e.g. `META_TYPE_OPAQUE` to avoid future collisions.

Reply:
Hmm, I thought (at least in Go), we'd be seeing `AssetMetaType_OPAQUE`?

Reply:
Yes, and that's why it's annoying in Go, since it becomes `AssetMetaType_META_TYPE_OPAQUE`. But on the gRPC level you get a startup panic if you try to load a package that has multiple enum value names that collide.

Reply:
Ah makes sense, will add the suffix so things won't break in other langs.